decidim-core/lib/decidim/asset_router.rb

```diff
@@ -22,7 +22,7 @@ def url(**options)
       if asset.is_a? ActiveStorage::Attached
         routes.rails_blob_url(asset.blob, **default_options.merge(options))
       else
-        routes.rails_representation_url(asset, **default_options.merge(options))
+        representation_url(**options)
```

Comment:
The entire content of this file should be reviewed after applying #[9597](https://github.com/decidim/decidim/pull/9597)

Reply:
Done.